.github/workflows/validate-frontend.yaml

```diff
@@ -0,0 +1,24 @@
+name: validate
```

Comment:
> validate
**Frontend Build and Lint** would be a better name for the workflow

Reply:
So the idea is that we can add jobs for the backend, etc in this wrokflow 
That's why kept the name  quite generic

Reply:
Frontend Build and Lint won't be good name as we are not utilising the built and linted code. We are just validating it